VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -621,13 +632,22 @@ qMRMLVirtualRealityView::qMRMLVirtualRealityView(QWidget* _parent) : Superclass(
 {
   Q_D(qMRMLVirtualRealityView);
   d->init();
+
 }
 
 // --------------------------------------------------------------------------
 qMRMLVirtualRealityView::~qMRMLVirtualRealityView()
 {
 }
 
+//------------------------------------------------------------------------------
+void qMRMLVirtualRealityView::registerModule(QWidget* widget, QIcon* icon) {
+  Q_D(qMRMLVirtualRealityView);
+
+  d->HomeWidget->setupModuleButton(widget, icon);
+
```

Comment:
Please remove this empty line and any other empty line that has no purpose (e.g. separate two blocks of code)